apiclient/container_start.go

```diff
@@ -0,0 +1,48 @@
+/*
+Copyright 2019 Thomas Weber
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+	http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
```

Comment:
This repo is now MPL'd and under HashiCorp, do you mind removing all of these headers?
```suggestion
```

Reply:
I changed the headers to comply with MPL-2, see https://www.mozilla.org/en-US/MPL/2.0/FAQ/